### DISCUSSION TITLE
Zellij creating `LEARN` folder

### DISCUSSION THREAD

**Discussion by steveny9911** [zellij-org/zellij#1630] [Q&A]
For some reason whenever a new Zellij session starts, the `LEARN` folder is created at user home. Is this an expected behaviour? If so, could I change the directory to somewhere else?

![image](https://user-images.githubusercontent.com/8460976/182291119-e2fe14f3-3521-41cc-b868-c438e4ae542d.png)

Ubuntu 20.04 on WSL2

**Comment by jaeheonji** [comment]
In general. This is not expected behavior.

That folder is shown as the socket folder of zellij.  And this folder usually refers to `$XDG_RUNTIME_DIR` .
* https://docs.rs/directories/0.8.5/directories/struct.ProjectDirs.html#method.runtime_dir

Therefore, I think there is a problem with the environment variable or the folder.
